src/plasma/eviction_policy.cc

```diff
@@ -19,8 +19,8 @@ class LRUCache {
   void add(const ObjectID &key, int64_t size) {
     auto it = item_map_.find(key);
     CHECK(it == item_map_.end());
-    item_list_.push_front(std::make_pair(key, size));
-    item_map_.insert(std::make_pair(key, item_list_.begin()));
+    item_list_.emplace_front(key, size);
+    item_map_.emplace(key, item_list_.begin());
```

Comment:
looks like this should work since we're using lists, but maybe add a comment that this relies on the fact that we're using lists so the iterators are not invalidated